File: src/frontend/src/app/components/atendimento/profissional/model/profissional-grid-dto.ts
export class ProfissionalGridDTO {
  id: string;
  pessoaNome: string;
  conselho: string;
  codigoConselho: string;
  tipoRemuneracao?: string;
  ativo?: boolean;
  deleted?: boolean;

  constructor(
    id: string,
    pessoaNome: string,
    conselho: string,
    codigoConselho: string,
    tipoRemuneracao?: string,
    ativo?: boolean,
    deleted?: boolean
  ) {
    this.id = id;
    this.pessoaNome = pessoaNome;
    this.conselho = conselho;
    this.codigoConselho = codigoConselho;
    this.tipoRemuneracao = tipoRemuneracao;
    this.ativo = ativo;
    this.deleted = deleted;
  }
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/atendimento/profissional/dto/ProfissionalGridDTO.java
package br.com.grupopipa.gestaointegrada.atendimento.profissional.dto;

import java.time.LocalDateTime;
import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Getter
@Setter
public class ProfissionalGridDTO implements GridDTO {
    private UUID id;
    private String pessoaNome;
    private String conselho;
    private String codigoConselho;
    private String tipoRemuneracao;
    private Boolean ativo;
    private LocalDateTime createdAt;
    private Boolean deleted;
}
